cartographer/mapping/2d/submap_2d_test.cc

```diff
@@ -79,14 +79,18 @@ TEST(Submap2DTest, TheRightNumberOfRangeDataAreInserted) {
     }
   }
   EXPECT_EQ(2, submaps.submaps().size());
-  int correct_num_range_data = 0;
+  int correct_num_finished_submaps = 0;
+  int num_unfinished_submaps = 0;
   for (const auto& submap : all_submaps) {
     if (submap->num_range_data() == kNumRangeData * 2) {
-      ++correct_num_range_data;
+      ++correct_num_finished_submaps;
+    } else if (submap->num_range_data() == kNumRangeData) {
```

Comment:
Move the condition into an `EXPECT_EQ`? Even if the actual `num_range_data()` is, this would still be an unfinished submap.

Reply:
Sorry I don't understand the second sentence, but I moved the condition into the `EXPECT_EQ`.